### DISCUSSION TITLE
Distance-Highet graph

### DISCUSSION THREAD

**Discussion by YBB8201** [IsraelHikingMap/Site#1926] [Ideas]
Hello
2 suggetions:
1.  add "accumulated hight" for every point the user stand on the graph.
2. add option for the user to choose the scale for X axis or for y axis - sale i mean distance between verticl mesh lines and hoizontal mesh lines

would like to hear your thoughts about it.
Yohai

![Capture](https://github.com/IsraelHikingMap/Site/assets/66330263/1902d976-f93e-4426-9a2e-95473feabed1)

**Comment by HarelM** [comment]
Hi,

Thanks for taking the time to write this down!
The graph automatically scales the x and y labels. In order to avoid cluttering, the current number of x and y labels is automatic.
I don't think any user should play with that, making it configurable will just complicate things for most users and will help very few if at all.
Regarding the accumulated height, you can click-drag on the graph to see accumulated height of a section of the graph. 
This is another example of features that are implemented without a good user experience and user do not know they can use them. We are trying to avoid adding more of those...

**Comment by YBB8201** [comment]
hi Harel,
about axis scales - I anderstand your approch
about the acumulated haighet - now i see i can use the click-drag - and it good. but if you can add the accumulated haight to the snippet, it will "save" the additional cick-drag operation. dont you think it will ease the use to get accumulated height (from start point) data?

**Reply by HarelM** [reply]
I don't see the added value in adding that and cluttering the current snippet window. Moreover, if adding accumulated height, the next request will be accumulated "fall".
When planning a route, you have a view that updates as you add more points to the route, in 99% of the cases it should be sufficient as far as I understand.

**Comment by YBB8201** [comment]
ok, got the point.
thanks!

**Reply by HarelM** [reply]
No problem, thanks for the input! I hope I didn't come of too aggressive, as this wasn't my intention.
